### ISSUE TITLE
Could we have a prop to refresh captcha without needing to trigger retry?

### ISSUE THREAD

**Issue by Daniel-Yonkov** [majid-amiri/react-client-captcha#3]
This feature will be suitable after for submission, if the data validation on the backend side is not correct. 
I.E. 
Login form submission, where the captcha maches but the credentials do not

**Comment by lavin-ds** [comment]
I had the same requirement, so did a workaround by adding a key to the component which is changed after every form submit. This forces the component to re-render

**Comment by majid-amiri** [comment]
> > I had the same requirement, so did a workaround by adding a key to the component which is changed after every form submit. This forces the component to re-render
> 
> Hmm, nice idea, will give it a try, maybe updating the documentation with this workaround till feature is deployed? @majid-amiri

Of course. Actually I'm going to add a separate props for manual retry functionality. Thanks for your comments.

**Comment by Daniel-Yonkov** [comment]
@majid-amiri  - Any progress on this?

**Comment by ajmal-n** [comment]
> I had the same requirement, so did a workaround by adding a key to the component which is changed after every form submit. This forces the component to re-render

can u please elaborate with an example code?
